app.js

```diff
@@ -0,0 +1,24 @@
+const { getAllTopics } = require('./controllers/topics-controller');
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+// Task 2 - 'GET /api/topics'
+app.get('/api/topics', getAllTopics);
+
+app.use((err, req, res, next) => {
+    if(!err.status && err.msg) {
+        res.status(err.status).send(err.msg);
```

Comment:
check this error handler, if the err.status and err.msg is falsey, then it will not be able to send anything i.e. res.send(undefined)